src/Aquifer.Common/Jobs/Publishers/TranslationPublisher.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Aquifer.Common.Jobs.Messages;
+
+namespace Aquifer.Common.Jobs.Publishers;
+
+public interface ITranslationPublisher
```

Comment:
I decided to do a naming change and make all services that publish messages to queues have `Publisher` at the end of the class name and all methods that do the publishing to have `Publish` at the beginning of the method name.  I'm planning to update existing publishers in a separate PR after this one merges (along with some other naming and directory changes).

Reply:
@MrGrinst, you're suggesting `ITranslationMessagePublisher`?  I don't object though I'd also want to change the subscribers to `MessageSubscriber` for consistency.